### PR TITLE
Upgrade sketches-go library

### PR DESCRIPTION
**Pull request by piochelepiotr**
New version is a bit more efficient at adding elements to sketches (Doesn't re-compute min and max indexable values).

### CODE REVIEW

**Review by brycekahle — APPROVED** [review]

File: .wwhrd.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ exceptions:
   - "golang.org/x/mod/..."
   - "golang.org/x/xerrors/..."
   - "github.com/stretchr/testify"
+  - "github.com/DataDog/sketches-go"
```
Comment (mx-psi) [inline .wwhrd.yml:25]: I don't think we want this here: the rest of exceptions are for tooling that is not really included in the Agent binary; this one is so it should be added to the licenses file.

**Review by mx-psi — CHANGES_REQUESTED** [review]

File: .wwhrd.yml (hunk 1 above)
Comment (piochelepiotr) [reply .wwhrd.yml:25]: 👋 I'm not sure I understand where you want me to add it?
The license for this package is not up to date. So the CI is failing. Do you know how else I can fix the CI?

Comment (mx-psi) [reply .wwhrd.yml:25]: You can revert commit c2679a8d61f392c65ab15748bdcd908b34fffbc7 and run `inv generate-licenses` to update the license list

Comment (piochelepiotr) [reply .wwhrd.yml]: Cool, looks like I run into an issue running `inv generate-licenses`:
```
vendoring dependencies
zsh:1: command not found: modvendor
```

I installed `go get github.com/goware/modvendor`

but then I run into:
```
zsh:1: command not found: wwhrd
```

etc,

Do you know what's the right way to setup this tool?

Comment (mx-psi) [reply .wwhrd.yml]: Looks like you need to [run `int install-tools`](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/agent_dev_env.md#installing-dependencies) :)

**Review by mx-psi — APPROVED** [review]

File: .wwhrd.yml (hunk 1 above)
Comment (piochelepiotr) [reply .wwhrd.yml]: Oh, great, thanks!

**Review by gbbr — APPROVED** [review]
🎉

**Pull request merged**